node/node_genesis.go

```diff
@@ -69,6 +70,10 @@ func (node *Node) SetupGenesisBlock(db ethdb.Database, shardID uint32, myShardSt
 	switch config.Network {
 	case config.Mainnet:
 		chainConfig = *params.MainnetChainConfig
+		foundationAddress := common.HexToAddress("0xE25ABC3f7C3d5fB7FB81EAFd421FF1621A61107c")
```

Comment:
not one1 address?

Reply:
This is created from nano ledger. It can be one address. but better using the old address for easy identification.